src/cpu/x64/gemm/gemm_driver.cpp

```diff
@@ -597,6 +597,7 @@ static dnnl_status_t gemm_kernel_driver(int ithr, dim_t m, dim_t n, dim_t k,
     if (mem_size > 0) {
         mem = (char *)malloc(mem_size, 128);
         if (!mem) return dnnl_out_of_memory;
+        memset(mem, 0, mem_size);
```

Comment:
Not sure if this is actually fixes the issue.
I mean with this change we will definitely get rid of valgrind error, but this is not what we really want
Possible concerns:
1) In general it is not required to default-initialize memory with some values (zeros in this case).
2) After this change MVN node will read zeros from this piece of memory. Are we sure that this is ok and will not affect the computation results?
3) That's great you have found that the root case is located in oneDNN code. We should try to check the latest oneDNN version and see if this issue is fixed there.